bazel/grpc_build_system.bzl

```diff
@@ -87,6 +89,11 @@ def grpc_cc_library(
     linkopts = if_not_windows(["-pthread"])
     if use_cfstream:
         linkopts = linkopts + if_mac(["-framework CoreFoundation"])
+    # Temporary hack for GRPC_USE_ABSL {
```

Comment:
if you say "temporary hack" you also need to add some more details, like this I don't know anything :-)

Reply:
I updated this a bit.